client/src/components/SentimentAnalysis/Helpers.js

```diff
@@ -8,12 +8,14 @@ const AnalysisInput = props => {
 
```

Comment:
Please add a JSDoc somewhere to this with background on what this is doing/why/general context.

Reply:
+1

Reply:
It's just a textarea so I thought it's already self-explainatory :-?

Reply:
I mean in reference to the entire doc. The title `helpers` doesn't really tell us what's in the code, and it's good style to just have a JSdoc giving an overview without having to dissect the code/what is being returned.